Lombiq.Analyzers/Build.props

```diff
@@ -63,7 +67,20 @@
     </ItemGroup>
   </Target>
 
-  <Target Name="Copy editorconfig" AfterTargets="AfterResolveReferences" Condition="'$(SolutionDir)' != '*Undefined*'">
+  <PropertyGroup Condition="'$(NuGetBuild)' == 'true'">
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <MSBuildTreatWarningsAsErrors>true</MSBuildTreatWarningsAsErrors>
+    <CodeAnalysisTreatWarningsAsErrors>True</CodeAnalysisTreatWarningsAsErrors>
+  </PropertyGroup>
+
+  <Target Name="Copy editorconfig"
+          AfterTargets="AfterResolveReferences"
+          BeforeTargets="EmbeddModuleAssets;OrchardCoreEmbedModuleAssets;Compile"
```

Comment:
You'll probably only need `Compile` here - the other two targets are Ochard-specific.